asset-schemas/ontology_info_schema.json

```diff
@@ -4,7 +4,7 @@
   "description": "A schema for the set of valid ontology reference files mapping to a CZ CellXGene Dataset Schema Versions",
   "type": "object",
   "patternProperties": {
-    "^[0-9]+\\.[0-9]+\\.[0-9]+$": {
+    "^[0-9]+\\.[0-9]+\\.[0-9]+(\\+.+)?$": {
```

Comment:
@nayib-jose-gloria i had to make a change here to allow `5.2.0+experimental` to be a valid version. usually for release candidates i've seen for various packages, even if version names generally follow semantic versioning, you might see something like `5.2.0-rc` so i think allowing for some sort of suffix seems reasonable.